src/components/ResponsiveGrid/ResponsiveGrid.tsx

```diff
@@ -0,0 +1,187 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import {
+	StandardProps,
+	findTypes,
+	omitProps,
+} from '../../util/component-types';
+import Resizer from '../Resizer/Resizer';
+
+const cx = lucidClassNames.bind('&-ResponsiveGrid');
+
+const { string, number, arrayOf } = PropTypes;
+
+interface IResponsiveGridCellProps extends StandardProps {}
+
+const Cell = (_props: IResponsiveGridCellProps): null => null;
+Cell.displayName = 'ResponsiveGrid.Cell';
+Cell.peek = {
+	description: `
+		Renders a \`<span>\` as the grid cell
```

Comment:
Should this say `<article>`, per line 77 in this file?

Reply:
yep, good catch!